fundamentals/middleware/problem-details-service/Program.cs

```diff
@@ -104,12 +102,15 @@
     await next(context);
     if (context.Response.StatusCode > 399)
     {
-        if (context.RequestServices.GetService<IProblemDetailsService>() is { } problemDetailsService)
+        if (context.RequestServices.GetService<IProblemDetailsService>() is
+                                                           { } problemDetailsService)
         {
             (string Detail, string Type) details = mathErrorFeature.MathError switch
             {
-                MathErrorType.DivisionByZeroError => ("The number you inputed is zero", "https://en.wikipedia.org/wiki/Division_by_zero"),
-                _ => ("Negative or complex numbers are not handled", "https://en.wikipedia.org/wiki/Square_root")
+                MathErrorType.DivisionByZeroError => ("The number you inputed is zero",
+                "https://en.wikipedia.org/wiki/Division_by_zero"),
+                _ => ("Negative or complex numbers are not handled",
+                "https://en.wikipedia.org/wiki/Square_root")
```

Comment:
I just split up a few lines so the code would show without a horizontal scroll bar on tablet sized devices.

Reply:
makes sense🤔